internal/service/cloudbackupschedule/resource_cloud_backup_schedule_test.go

```diff
@@ -775,20 +775,20 @@ func configExportPolicies(info *acc.ClusterInfo, policyName, roleName, bucketNam
     }
 
     resource "mongodbatlas_cloud_provider_access_setup" "setup_only" {
-        project_id      = %[2]s
+        project_id      = %[2]q
```

Comment:
it's ok to do quotes here?

Reply:
See comment above: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2426#discussion_r1679179283